src/presentation/ui/page/mappings.templ

```diff
@@ -0,0 +1,338 @@
+				@componentForm.SubmitButton("Create mapping", "ph-plus-square", "openCreateMappingModal()", false, false)
+			</div>
+		</div>
+		<div id="mappings-tables" class="card w-full">
+			@MappingsRegularFormTable(vhostsWithMappings)
+		</div>
+		@CreateVhostModal()
+		@componentStructural.DeleteModal(
+			"isDeleteVhostModalOpen", "closeDeleteVhostModal()", "deleteVhostElement()",
+			"virtualHost.hostname", "virtualHost.hostname",
+		)
+		@CreateMappingModal(vhostsWithMappings)
+		@CreateMappingFromVhostModal(vhostsWithMappings)
+		@componentStructural.DeleteModal(
+			"isDeleteMappingModalOpen", "closeDeleteMappingModal()", "deleteMappingElement()",
+			"mapping.path", "mapping.id",
+		)
+	</div>
+}
+
+templ MappingsRegularFormTable(vhostsWithMappings []dto.VirtualHostWithMappings) {
+	<table
+		id="vhosts-with-mappings-regular-form-table"
+		hx-get="/mappings/"
+		hx-trigger="submit from:form delay:500ms, click from:button#delete-element delay:500ms"
+		hx-select="#mappings-tables"
+		hx-target="#mappings-tables"
+		hx-swap="outerHTML transition:true"
+		class="w-full table-auto border-collapse rounded-md transition-all duration-300 ease-in-out"
+	>
+		<div class="bg-os-800 p-4 rounded-md">
+			for _, vhostWithMappings := range vhostsWithMappings {
+				<div class="flex bg-os-500 mb-4 p-4 rounded">
+					<!-- Vhosts Column -->
+					<div class="flex bg-os-800 p-4 w-3/8 min-h-56 flex-col rounded">
+						<div class="flex justify-between items-center">
+							<span class="text-xl">{ vhostWithMappings.Hostname.String() }</span>
+							<div class="flex space-x-2">
+								@componentStructural.RoundButton("ph-plus", "speedia-500", "speedia-300", "openCreateMappingFromVhostModal('"+vhostWithMappings.Hostname.String()+"')", "Create mapping", "")
+								if vhostWithMappings.Type != "primary" {
+									@componentStructural.RoundButton("ph-trash", "red-800", "red-600", "openDeleteVhostModal('"+vhostWithMappings.Hostname.String()+"')", "Delete virtual host", "")
+								}
+							</div>
+						</div>
+						<div class="flex space-x-3 mt-4">
+							@componentStructural.TagWithIcon("ph-folder-open", vhostWithMappings.RootDirectory.String(), "speedia-300")
+							@componentStructural.TagWithIcon("ph-file-text", vhostWithMappings.Type.String(), "speedia-300")
+						</div>
+					</div>
+					<div class="w-4"></div>
+					<!-- Mappings Column -->
+					<div class="bg-os-800 p-4 flex-col rounded w-1/1">
+						for _, mapping := range vhostWithMappings.Mappings {
+							@componentMappings.MappingsInfos(
+								mapping, vhostWithMappings.RootDirectory,
+								"openDeleteMappingModal("+mapping.Id.String()+", '"+mapping.Path.String()+"')",
+							)
+						}
+					</div>
+				</div>
+			}
+		</div>
+	</table>
+}
+
+templ VirtualHostRegularForm() {
+	<form
+		hx-post="/api/v1/vhosts/"
+		hx-indicator="#loading-overlay"
+		hx-swap="none"
+	>
```

Comment:
Isn't this a Create operation? Why is this called RegularForm?

Reply:
Changed to CreateVhostForm.